amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -707,5 +728,23 @@ void ABLBoundaryPlane::impl_buffer_field(
                                              k - v_offset[2], n));
         });
 }
+
+// True if box intersects the boundary
```

Comment:
```suggestion
//! True if box intersects the boundary
```

Make doxygen friendly